clients/python/lakefs_client/api/objects_api.py

```diff
@@ -935,6 +935,7 @@ def __upload_object(
 
             Keyword Args:
                 storage_class (str): [optional]
+                if_none_match (str): Currently supports only \"*\" to allow uploading an object only if one doesn't exist yet.. [optional]
```

Comment:
note that we can avoid the '..' at the end if the description value will not end with '.'